### PR TITLE
Revert "misc readme updates"

### PR DESCRIPTION
**Pull request by rlball**
Reverts pcelvng/go-config#2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -408,6 +389,7 @@ func main() {
     config.DisableTOML()
```
Comment (jbsmith7741) [inline README.md:389]: What do you think of functional options 
``` go 
config.Load(&config{}, config.OnlyJSON())
```

Comment (rlball) [reply README.md:389]: Not a bad idea. Let me think about it. I like how compact that is. I want to make sure it's intuitive enough that the user will either remember it after viewing the docs and easily see what's going on if it's someone who hasn't used the lib before. Perhaps both could be used?

Comment (rlball) [reply README.md:389]: Maybe chaining isn't out of the question either?

```go
config.Load(&cfg{}).OnlyJSON()
```

Comment (rlball) [reply README.md:389]: If the options are functions on the package or chained methods then someone with intelliSense could see the options without needing to consult the docs too much.

**Pull request closed without merging**